pyvo/dal/sia.py

```diff
@@ -903,7 +903,7 @@ def suggest_dataset_basename(self):
             out = re.sub(r'\s+', '_', out.strip())
         return out
 
-    def suggest_extension(self, *, default=None):
+    def suggest_extension(self, *, default='img'):
```

Comment:
Just curious why `img`? I've noticed same assumption in `suggest_dataset_basename`.